### PR TITLE
Add a scroll down arrow

### PR DESCRIPTION
**Pull request by viciouskitten**
Added a scroll down arrow with nice animation on hover

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,7 +63,9 @@ <h2 class="text-dec">KamandPrompt</h2>
             </div>
           </div>
       </div>
-      <div class="small-text no-style bottom-center"><a href="#about">Scroll down to know more</a></div>
+      <div class="small-text no-style bottom-center">
+        <a class="scroll-down" href="#about">Scroll down</a>
```
Comment (abhigyank) [inline index.html]: @viciouskitten Can you change the text to `Scroll down to know more` ?

**Review by abhigyank — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (viciouskitten) [reply index.html]: Would you accept "Scroll down to learn more?" i think it sounds better. Happy to change it back if you don't agree though :)

Comment (abhigyank) [reply index.html]: Cool. That sounds fine.

**Review by abhigyank — APPROVED** [review]

**Comment by abhigyank** [comment]
Thanks @viciouskitten  :rocket:

**Pull request merged**